package.json

```diff
@@ -58,11 +56,11 @@
     "index.js"
   ],
   "dependencies": {
-    "cf-client": "^1.0.0",
-    "hubot-cf-convenience": "0.0.4",
+    "cf-client": "latest",
```

Comment:
@aeweidne Do you think we should scrap this dependency and rely on `hubot-cf-convenience` to bring in this dependency?